.storybook/main.js

```diff
@@ -7,19 +7,37 @@
 
 module.exports = {
   stories: ['./*.stories.mdx', '../packages/**/demo/*.stories.mdx'],
+  staticDirs: ['../packages/bedrock/dist'],
+  framework: {
+    name: '@storybook/react-webpack5',
+    options: {}
+  },
+  docs: {
+    autodocs: true
+  },
   addons: [
-    { name: '@storybook/addon-essentials', options: { actions: false } },
     {
-      name: '@storybook/addon-postcss',
-      options: { postcssLoaderOptions: { implementation: require('postcss') } }
+      name: '@storybook/addon-essentials',
+      options: {
+        actions: false
+      }
+    },
+    {
+      name: '@storybook/addon-styling',
+      options: {
+        postCss: {
+          implementation: require('postcss')
+        }
+      }
     }
   ],
-  core: {
-    builder: 'webpack5'
-  },
   webpackFinal: (config) => {
     config.output.hashFunction = 'xxhash64';
 
     return config;
-  }
+  },
+  managerHead: (head) => `
+    ${head}
+    <link rel="icon" href="https://zendeskgarden.github.io/favicons/garden/favicon.ico" />
+  `
```

Comment:
Can we revert and keep this aligned with https://github.com/zendeskgarden/react-containers/tree/main/.storybook